include/sdlmain.h

```diff
@@ -203,39 +288,41 @@ struct SDL_Block {
 	bool mute_when_inactive  = false;
 	bool pause_when_inactive = false;
 
-	SDL_Rect clip_px = {0, 0, 0, 0};
-	SDL_Window *window = nullptr;
-	SDL_Renderer *renderer = nullptr;
+	SDL_Rect draw_rect_px     = {};
+	SDL_Window* window        = nullptr;
+	SDL_Renderer* renderer    = nullptr;
 	std::string render_driver = "";
-	int display_number = 0;
+	int display_number        = 0;
 
 	struct {
-		SDL_Surface *input_surface = nullptr;
-		SDL_Texture *texture = nullptr;
-		SDL_PixelFormat *pixelFormat = nullptr;
+		SDL_Surface* input_surface   = nullptr;
+		SDL_Texture* texture         = nullptr;
+		SDL_PixelFormat* pixelFormat = nullptr;
 	} texture = {};
 
 	struct {
 		present_frame_f* present      = present_frame_noop;
 		update_frame_buffer_f* update = update_frame_noop;
 		FrameMode desired_mode        = FrameMode::Unset;
 		FrameMode mode                = FrameMode::Unset;
-		double period_ms    = 0.0; // in ms, for use with PIC timers
+
+		// in ms, for use with PIC timers
+		double period_ms      = 0.0;
 		float max_dupe_frames = 0.0f;
-		int period_us       = 0; // same but in us, for use with chrono
+
+		// same but in us, for use with chrono
+		int period_us       = 0;
 		int period_us_early = 0;
-		int period_us_late    = 0;
+		int period_us_late  = 0;
```

Comment:
I know this is just a formatting change, but should these be `int64_t` if they are microseconds, and used with chrono, since we also updated `GetTicksxx` to return int64_t's?

Reply:
`int` is OK, because these are boundary values calculated from the video frame period.
- The largest late period is ~60,000 us if someone overrides the DOS rate to the 24 Hz min (producing the largest late frame period)
- The smallest early period is ~550 us if someone overrides the DOS rate all the way to the 1000 Hz max (producing the smallest early frame period).

Reply:
Yeah and I'm reluctant to add more to this PR anyway 😅 So no "while you're at it, can you also..." type of comments now, please 😄